index.bs

```diff
@@ -388,6 +388,7 @@ are returned to the caller when a new credential is created, or a new assertion
     interface ScopedCredential : Credential {
         readonly attribute ArrayBuffer           rawId;
         readonly attribute AuthenticatorResponse response;
+        readonly attribute AuthenticationExtensions clientExtensionResults;
```

Comment:
Could you add a description of this field after the description of [response](https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/selfissued/webauthn/mbj-extensions-connective-tissue/index.bs#dom-scopedcredential-response)? Those descriptions are non-normative, but they help folks understand the spec. Something like
> This attribute contains the [=client extension output=] produced by the extension's [=client extension processing=]. The [=authenticator extension output=] is returned through the {{AuthenticatorAttestationResponse/attestationObject|AuthenticatorAttestationResponse.attestationObject}} or {{AuthenticatorAssertionResponse/authenticatorData|AuthenticatorAssertionResponse.authenticatorData}} depending on the call that produced this credential.